domain-models-maven-plugin/src/main/java/org/folio/rest/tools/DomainModelsMojo.java

```diff
@@ -103,7 +103,10 @@ public void execute() {
         generateInterfaces();
         ModuleNameWriter.writeModuleNameClass(project);
       }
-      ClientGenerator.generate(project.getBasedir().getAbsolutePath(), generateClients);
+      if (generateClients) {
+        int sz = ClientGenerator.generate(project.getBasedir().getAbsolutePath());
+        System.err.println("Client classes generated: " + sz);
```

Comment:
Why `System.err` not `System.out`?

Reply:
Other print statements in the plugin was err. Can you comment on that @julianladisch ? Is print statements the best to report info for a plugin?

Reply:
`System.err` should be used for error logging, `System.out` should be used for non-error logging.
This might be used for different colors when displaying the output of the build process.
The existing statements (10 x System.err, 3 x System.out) are inconsistent.